.gitignore

```diff
@@ -18,3 +18,4 @@
 
 .DS_Store
 .env
+config/database.yml
```

Comment:
By default Postgres creates its first superuser account based on the name of the system user that called initdb when installed After that the application user and database user are independent, and you can even rename the database superuser if you want.

http://stackoverflow.com/questions/1471571/how-to-configure-postgresql-for-the-first-time